### PR TITLE
Issue 256

### LINKED ISSUE #256: Should all DGGS support (functions) be in their own Conformance Classes?
Non-topological query functions are split into 4 Conformance Classes:

1. SF Functions
2. Non-SF Functions
3. SF Functions - DGGS
4. Non-SF Functions - DGGS

The SF / non-SF split is not relevant to this discussion.

The traditional geometries/DGGS split is due to:

> The requirements to support non-topological query functions for DGGS geometry literals are separated from the requirements to support them for traditional geometry literals as it is expected that implementing these functions for DGGS literals will be significantly more difficult. This is due to the novelty of DGGS literals and thus the lack of existing software libraries for their manipulation. 

Should all GeoSPARQL functions support be split into traditional/DGGS Conformance Class pairs?

### PR DESCRIPTION
**Pull request by nicholascar**
I have added a new Conformance Class just for DGGS and remove DGGS requirements from the other CCs.

I've also tidied up word references to Requirements & Conformance Classes and r-issued figure 2 (CC diagram) as there were inconsistencies.

I've also de-duplicated the Res RDF file by using the already present `dcterms:requires` mecahnism for 1.1 CCs to import 1.0 CCs.

Closes #256

### CODE REVIEW

**Review by situx — APPROVED** [review]
Looks good to me.
However, the requirements need to be adjusted for DGG URIs to be added in a separate pull request.
Also please check again for some typos.
Also: We should merge this in before I finalize my service descriptions, as requirements changed

File: 1.1/reqs.ttl [hunk 1]
```diff
@@ -1,980 +1,1157 @@
...
+    skos:definition "Verify that queries involving geo:geoJSONLiteral values return the correct result for a test dataset"@en ;
+    skos:prefLabel "GeoSPARQL 1.1 Conformance Test: GeoJSON Literal"@en ;
+.
+
+conf11gx:geojson-literal-empty
+    a spec:ConformanceTest ;
+    spec:testPurpose "check conformance with this requirement"@en ;
+    spec:testType spec:Capabilities ;
+    rdfs:seeAlso reqs11gx:geojson-literal-empty ;
+    skos:definition "Verify that queries involving geo:kmlLiteral values without an explicit encoded SRS IRI return the correct result for a test dataset"@en ;
```
Comment (mperry455) [inline 1.1/reqs.ttl:634]: Should be "Verify that queries involving empty geo:geoJSONLiteral values return the correct result for a test dataset"

File: 1.1/spec/05-Part-02.adoc [hunk 2]
```diff
@@ -1,48 +1,29 @@
...
+Conformance with this specification shall be checked using all the relevant tests specified in <<Annex A - Abstract Test Suite (normative)>>. The framework, concepts, and methodology for testing, and the criteria to be achieved to claim conformance are specified in _ISO 19105: Geographic information — Conformance and Testing_ <<ISO19105>>.
 
-This document establishes several requirements classes and corresponding conformance classes (a conformance class is a set of tests for each requirement in a requirements class). Any GeoSPARQL implementation claiming conformance with one of the conformance classes shall pass all the tests in the associated abstract test suite.
+This document establishes many individual Requirements and Conformance Classes which contain tests for one or more Requirements. GeoSPARQL implementations need not conform to all Conformance Classes but must state which individual ones the do conform to. GeoSPARQL implementations claiming conformance to a Conformance Classes must pass all the tests defined for it in the <<Annex A - Abstract Test Suite (normative)>>.
```
Comment (mperry455) [inline 1.1/spec/05-Part-02.adoc:4]: "ones the do conform to" -> "ones they do conform to"

File: 1.1/spec/10-Part-07.adoc [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 
-The following parameter is defined for the _Topology Vocabulary Extension_ requirements class.
+The following parameter is defined for the _Topology Vocabulary Extension_ Requirements.
 
 *relation_family*: Specifies the set of topological spatial relations to support.
 
 === Simple Features Relation Family
 
-This clause defines requirements for the _Simple Features_ relation family.
+This clause defines Requirement for the _Simple Features_ relation family.
```
Comment (mperry455) [inline 1.1/spec/10-Part-07.adoc:15]: Requirements

File: 1.1/spec/10-Part-07.adoc [hunk 4]
```diff
@@ -49,7 +49,7 @@ L/A+`; `+(0********) for L/L+`
 
 === Egenhofer Relation Family
 
-This clause defines requirements for the 9-intersection model for binary topological relations (_Egenhofer_) relation family. Consult references <<FORMAL>> and <<CATEG>> for a more detailed discussion of _Egenhofer_ relations.
+This clause defines Requirement for the 9-intersection model for binary topological relations (_Egenhofer_) relation family. Consult references <<FORMAL>> and <<CATEG>> for a more detailed discussion of _Egenhofer_ relations.
```
Comment (mperry455) [inline 1.1/spec/10-Part-07.adoc:52]: Requirements

File: 1.1/spec/11-Part-08.adoc [hunk 5]
```diff
@@ -1,6 +1,6 @@
 == Geometry Extension
 
-This clause establishes the _Geometry Extension_ parameterized requirements class, with IRI `/req/geometry-extension`, which has a single corresponding conformance class _Geometry Extension_, with IRI `/conf/geometry-extension`. This requirements class defines a vocabulary for asserting and querying information about geometry data, and it defines query functions for operating on geometry data.
+This clause establishes the _Geometry Extension_ parameterized Requirements with base IRI `/req/geometry-extension`, which have a single corresponding conformance class _Geometry Extension_, with IRI `/conf/geometry-extension`. These Requirements defines a vocabulary for asserting and querying information about geometry data, and defines query functions for operating on geometry data.
```
Comment (mperry455) [inline 1.1/spec/11-Part-08.adoc:3]: These Requirements define ..., and define ...

File: 1.1/spec/14-Part-11.adoc [hunk 6]
```diff
@@ -1,6 +1,6 @@
...
+This clause establishes the _Query Rewrite Extension_ parameterized Requirements with base IRI `/req/query-rewrite-extension`, which has a single corresponding conformance class _Query Rewrite Extension_, with IRI `/conf/query-rewrite-extension`. These Requirements defines a set of RIF rules <<RIF>> that use topological extension functions defined in Clause 9 to establish the existence of direct topological predicates defined in Clause 7. One possible implementation strategy is to transform a given query by expanding a triple pattern involving a direct spatial predicate into a series of triple patterns and an invocation of the corresponding extension function as specified in the RIF rule.
```
Comment (mperry455) [inline 1.1/spec/14-Part-11.adoc:3]: These Requirements define

**Review by mperry455 — APPROVED** [review]

**Pull request merged**